frontend/rust-lib/flowy-database/src/services/field/type_options/date_type_option/date_tests.rs

```diff
@@ -72,70 +72,70 @@ mod tests {
     }
   }
 
-  #[test]
-  fn date_type_option_different_time_format_test() {
-    let mut type_option = DateTypeOptionPB::default();
-    let field_type = FieldType::DateTime;
-    let field_rev = FieldBuilder::from_field_type(&field_type).build();
-
-    for time_format in TimeFormat::iter() {
-      type_option.time_format = time_format;
-      match time_format {
-        TimeFormat::TwentyFourHour => {
-          assert_date(
-            &type_option,
-            1653609600,
-            None,
-            "May 27,2022 00:00",
-            true,
-            &field_rev,
-          );
-          assert_date(
-            &type_option,
-            1653609600,
-            Some("9:00".to_owned()),
-            "May 27,2022 09:00",
-            true,
-            &field_rev,
-          );
-          assert_date(
-            &type_option,
-            1653609600,
-            Some("23:00".to_owned()),
-            "May 27,2022 23:00",
-            true,
-            &field_rev,
-          );
-        },
-        TimeFormat::TwelveHour => {
-          assert_date(
-            &type_option,
-            1653609600,
-            None,
-            "May 27,2022 12:00 AM",
-            true,
-            &field_rev,
-          );
-          assert_date(
-            &type_option,
-            1653609600,
-            Some("9:00 AM".to_owned()),
-            "May 27,2022 09:00 AM",
-            true,
-            &field_rev,
-          );
-          assert_date(
-            &type_option,
-            1653609600,
-            Some("11:23 pm".to_owned()),
-            "May 27,2022 11:23 PM",
-            true,
-            &field_rev,
-          );
-        },
-      }
-    }
-  }
+  // #[test]
```

Comment:
I see this is merged, but a few tests are commented out.

Should there be a todo to fix these, or should they be completely removed?